libcodechecker/server/product_db_access_handler.py

```diff
@@ -171,9 +311,17 @@ def addProduct(self, product):
         """
         Add the given product to the products configured by the server.
         """
+        self.__require_permission([permissions.SUPERUSER])
 
+        session = None
```

Comment:
Why do we set the session here to None? We will not use it until the line `440`.